decidim-accountability/app/permissions/decidim/accountability/admin/permissions.rb

```diff
@@ -10,6 +10,7 @@ def permissions
           permission_action.allow! if can_perform_actions_on?(:result, result)
           permission_action.allow! if can_perform_actions_on?(:status, status)
           permission_action.allow! if can_perform_actions_on?(:timeline_entry, timeline_entry)
+          permission_action.allow! if can_perform_actions_on?(:projects_import, projects_import)
```

Comment:
Please take reference from the `:import_proposals` permission action within `Decidim::Budgets::Admin::Permissions` for this check.